kubeflow/weaveflux/prototypes/weaveflux.jsonnet

```diff
@@ -4,7 +4,7 @@
 // @shortDescription A Flux meets Kubeflow
 // @param name string Name to give to each of the components
 // @optionalParam giturl string null Your default git URL.
-// @optionalParam serviceType string ClusterIP The service type for WeaveFlux.
+// @optionalParam serviceType string NodePort The service type for WeaveFlux.
```

Comment:
I think this is the same as @jlewi point before. We should default to ClusterIP.